src/app/shared/shared.module.ts

```diff
@@ -87,6 +88,7 @@ import { OffCampusRecordDetailComponent } from './components/off-campus-record-d
     MatSnackBarModule,
 
     OffCampusRecordDetailComponent,
+    SharedRecordListComponent,
```

Comment:
Since the component is declared in `SharedModule`, we don't need to import it, what `import` does is to import other modules and then we can use components, pipes or others `exported` from that module in our module. In your case, the `SharedRecordListComponent` is already in the `declarations` section, then all components in `SharedModule` will be able to use this component, and if you want other modules can use this component, just add it to `exports`, so other components in other modules can use your component as long as their modules have imported your `SharedModule`. Please also fix `OffCampusRecordDetailComponent` above this line.